bigquery/deploy/README.md

```diff
@@ -33,7 +33,9 @@
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
     * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
-    * Expand `More` and select the service account you just created.
+    * Expand `More`
+      * Select the service account you just created
+      * `Set access for each API` -> Change `BigQuery` to enabled
```

Comment:
This step was intentionally removed, after a discussion about access scopes being a legacy method of authentication. We shouldn't need to set access scopes to be able to run our indexers, and I don't think this actually works with a non-default service account. Try following these instructions with a non-default service account and notice that the access scopes section disappears when you select the non-default service account.

Reply:
Also: feel free to split this documentation change out from the rest of this PR while we work this out if you want me to LGTM the refactor to unblock any other work

Reply:
Ah, I see what happened.
- Access scopes are orthogonal to IAM. The indexer service account needs BigQuery access scope in order to read BQ tables, regardless of what IAM permissions are in place.
- In the create cluster UI, when you select non-default service account, the access scope setting automatically changes to "All scopes" and can't be changed
- This can be confirmed by looking at a GKE VM in GCS. "Cloud API access scopes" is set to "Allow full access to all Cloud APIs".

Reverted README.